services/workflows-service/src/traces/tracer.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  SEMRESATTRS_SERVICE_VERSION,
+  SEMRESATTRS_SERVICE_NAME,
+} from '@opentelemetry/semantic-conventions';
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
+import { Resource } from '@opentelemetry/resources';
+import { trace } from '@opentelemetry/api';
+import * as opentelemetry from '@opentelemetry/sdk-node';
+import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
+import { dockerCGroupV1Detector } from '@opentelemetry/resource-detector-docker';
+import { B3Propagator } from '@opentelemetry/propagator-b3';
+import { version } from '../../package.json';
+import { WinstonInstrumentation } from '@opentelemetry/instrumentation-winston';
+import { NestInstrumentation } from '@opentelemetry/instrumentation-nestjs-core';
+import { AwsInstrumentation } from '@opentelemetry/instrumentation-aws-sdk';
+import { ExpressInstrumentation } from '@opentelemetry/instrumentation-express';
+import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
+import { PrismaInstrumentation } from '@prisma/instrumentation';
+
+const traceExporter = new OTLPTraceExporter({
+  // port configured in the Collector config, defaults to 4317
+  url: 'http://localhost:4318/v1/traces',
```

Comment:
Move to an env var

Reply:
there is existing env var which in used on this library already,
`OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`